omspy/models.py

```diff
@@ -168,7 +168,7 @@ def create(self, seconds: float) -> pendulum.DateTime:
         Lock the create_order function for the given number of seconds
         """
         seconds = min(seconds, self.max_order_creation_lock_time)
-        self._creation_lock_till = pendulum.now(tz=self.timezone).add(seconds=seconds)
+        self._creation_lock_till = pendulum.now(tz=self.timezone).add(seconds=int(seconds))
```

Comment:
seconds could be in float. we are giving a choice to user to specify seconds as float so shouldn't force this to int

Reply:
Could you make this change or can I change it at my end?

Reply:
the add method invoked expects int for its seconds parameter - please check. Ideally you should change upstream code to use int instead of float (or don't use pendulum api)

Reply:
Yes, pendulum expects int as type and the right thing would be to change the type.
But if the user wants to pass 1.5 he should pass them necessarily as 1 second and 500000 microseconds which looks a bit too much on part of the user. 
pendulum does convert automatically 1.5 seconds to 1 second and 500000 microseconds
Any suggestion on your part?

Reply:
We should necessarily strip the decimal part and convert it into microseconds.